src/resolvers/PhoneNumber.ts

```diff
@@ -2,7 +2,7 @@ import { GraphQLScalarType } from 'graphql';
 import { GraphQLError } from 'graphql/error';
 import { Kind } from 'graphql/language';
 
-const PHONE_NUMBER_REGEX = /^\+?[1-9]\d{1,14}$/;
```

Comment:
Thanks for keeping the `\+` as non-optional. I think it would have been a breaking change otherwise, especially since many parsing libraries fail without it.

Reply:
Yes, that's why I made it required :)